frontend/constants/index.ts

```diff
@@ -0,0 +1,2 @@
+// max length for an input string
+export const MAX_INPUT_STRING_LENGTH=32
```

Comment:
```suggestion
export const MAX_INPUT_STRING_LENGTH=32

```

Adding a newline at EOF